db/index.js

```diff
@@ -0,0 +1,54 @@
+var videos = require('./videoData.js')
+
+// console.log(videos)
+
+var mongoose = require('mongoose');
+mongoose.connect('mongodb://localhost/media', { useNewUrlParser: true, useUnifiedTopology: true });
+
+var db = mongoose.connection;
+
+db.on('error', function (err) {
+  console.log('mongoose connection error: ', err);
+});
+
+db.once('open', function () {
+  console.log('mongoose connected successfully');
+});
+
+var mediaSchema = mongoose.Schema({
+  id: {type: Number, unqiue: true},
```

Comment:
Typo: unique. Do you have a linter set up? HR recommends this: https://github.com/hackreactor-labs/eslint-config-hackreactor

Reply:
Point taken. I just put the linter in my package.json